crates/subspace-farmer/src/bin/subspace-farmer/commands/bench.rs

```diff
@@ -226,6 +226,25 @@ pub(crate) async fn bench(
         write_pieces_size as f64 / 1000. / 1000. / took.as_secs_f64()
     );
 
+    if !skip_recommitments_bench {
```

Comment:
Double negative isn't very nice, can we have `recommitments_bench` and default it to `true` in CLI?

Reply:
It would look weird in cli. You generally assert something via a boolean flag, if you omit it then you implicitly set false.

Reply:
I can negate it in the main file though

Reply:
Then we can have `no-recommitments` flag, but propagate it down the app as non-negative (BTW, I suspect Clap knows how to handle `no-` prefixes).